app/controllers/rademade_admin/model_controller.rb

```diff
@@ -31,14 +31,15 @@ def create
       render_record_errors e
     end
 
-    def update
-      saver = RademadeAdmin::Saver.new(model_info, params)
-      saver.find_model
-      authorize! :update, saver.item
-      saver.set_data
-      before_update saver.item
-      saver.save_item
-      success_update saver.item
+    def update(&block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `update(*)` if you want the method to accept any arguments but don't care about them.